app/views/application/_context_navbar.erb

```diff
@@ -11,13 +11,17 @@
 
   <%= tag.div class: "ml-auto d-flex flex-column flex-md-row justify-content-between" do %>
 
-    <%= link_to({ controller: controller.controller_name, action: :new },
-                { class: "mx-4 mr-lg-5 btn nav-item nav-link btn-link",
-                  id: "create_link" }) do -%>
-          <%= tag.span class: "text-nowrap" do -%>
-            <%= tag.span :CREATE.t, class: "mr-2" -%>
-            <%= tag.span class: "fas fa-edit" -%>
-          <% end %>
+    <% if Rails.application.routes.url_helpers.method_defined?(
+            "new_#{controller.controller_name.singularize}_path"
+    ) %>
```

Comment:
This fixes a bug. The partial throws an error for Users, as there is no `:new` action. 
I wonder if -- in the long run -- there should be one; seems simpler to have `users` be a resource with the full set of actions, rather than calling methods like `new` and `destroy` from AccountController. But that means we have to take extra steps to hide those actions and/or make them unavailable to non-admins.